stroyprombeton/views/catalog.py

```diff
@@ -187,18 +187,25 @@ def get_context_data(self, **kwargs):
         }
 
 
-# @todo #558:30m  Create custom page for series matrix page.
-#  It will provide page and seo features: h1, content, metadata and so on.
 def series_matrix(request):
-    series = (
-        models.Series.objects
-        .filter(page__is_active=True)
-        .order_by('name')
-    )
+    page = CustomPage.objects.get(slug='series')
+    # @todo #601:60m  Make Page to use db based url name.
+    #  Now we can't use custom url_name for CustomPage.
```

Comment:
It is little bit unclear. What wrong with current implementation? URL of an instance of `CustomPage` is resolved with help of its unique `slug` field and it seems enough

You just should define [custom_page_url](https://github.com/fidals/stroyprombeton/blob/master/stroyprombeton/urls.py#L38) and `get_absolute_url` will work

Reply:
@artemiy312 , you are right. Series matrix is just common custom page as `/gbi/` for example.
Fixed it